src/main/java/nl/tudelft/contextproject/model/entities/EntityType.java

```diff
@@ -27,7 +27,8 @@
 	
 	TREASURE("Treasure", 			0,  Treasure::loadEntity),
 	WALLFRAME("WallFrame", 			0,  WallFrame::loadEntity), 
-	INVISIBLE_WALL("InvisibleWall",	0,	InvisibleWall::loadEntity);
+	INVISIBLE_WALL("InvisibleWall",	0,	InvisibleWall::loadEntity), 
+	DAMAGED_WALL("DamagedWall",		0,	DamagedWall::loadEntity);
```

Comment:
if you change the webId for DamagedWall to 12 it will show up on the webinterface